database/sql/src/main/scala/cromwell/database/slick/MetadataSlickDatabase.scala

```diff
@@ -100,17 +100,21 @@ trait MetadataSlickDatabase extends MetadataSqlDatabase {
     }
   }
 
-  def refreshMetadataSummaries(startId: Long, startTimestamp: Option[Timestamp],
+  def refreshMetadataSummaries(startTimestamp: Option[Timestamp],
                                key1: String, key2: String, key3: String, key4: String,
                                buildUpdatedSummary: (Option[WorkflowMetadataSummary], Seq[Metadatum]) =>
                                  WorkflowMetadataSummary)
-                              (implicit ec: ExecutionContext): Future[Seq[Metadatum]] = {
+                              (implicit ec: ExecutionContext): Future[Long] = {
     val action = for {
+      startIdOption <- getSummaryStatusMaximumId("WORKFLOW_METADATA_SUMMARY", "METADATA_JOURNAL")
```

Comment:
I don't know if it's possible, but is there any way we can reference the name of the table through the ***Component class to avoid copy / pasting it ?